authentication/exception.py

```diff
@@ -0,0 +1,9 @@
+
+
+class DuplicateEmailError(Exception):
+
+    def __init__(self, msg):
+        self.msg = msg
```

Comment:
Message or exception should explicit instead of dynamic.

Reply:
Done.